game/world/objects/item.cpp

```diff
@@ -204,12 +205,19 @@ bool Item::isMulti() const {
   }
 
 bool Item::isSpellShoot() const {
+  // Whether a spell is a projectile is hardcoded in vanilla, see
```

Comment:
In link you provided:
```c++
  'bIsProjectile' TRUE (if visual effect name doesn't contain "_SPREAD"):
    38, 87
  'bIsProjectile' TRUE (always):
    2, 6, 19, 20, 23, 26, 30, 33, 34, 36, 43, 71, 72, 73, 80, 81, 82, 86
  'bIsProjectile' default is FALSE.
```

Can't find `47` there. Am I missing something?

Reply:
These are for G2, G1 has different ids. Except Stormfist `TARGET_COLLECT_ALL_FALLBACK_NONE` is used for icewave, firerain and uriziel rune which are all aoe effects.

Originally I had another source which gave only the hardcoded fact, but couldn't find it anymor. Found one from a dev instead :)